core_api/src/routes/chat.py

```diff
@@ -50,12 +50,16 @@
 )
 
 ROUTE_RESPONSES = {
-    "info": ChatPromptTemplate.from_template(INFO_RESPONSE),
-    "ability": ChatPromptTemplate.from_template(ABILITY_RESPONSE),
-    "coach": ChatPromptTemplate.from_template(COACH_RESPONSE),
-    "gratitude": ChatPromptTemplate.from_template("You're welcome!"),
-    "summarisation": ChatPromptTemplate.from_template("You are asking for summarisation - route not yet implemented"),
-    "extract": ChatPromptTemplate.from_template("You asking to extract some information - route not yet implemented"),
+    ChatRouteEnum.info: ChatPromptTemplate.from_template(INFO_RESPONSE),
+    ChatRouteEnum.ability: ChatPromptTemplate.from_template(ABILITY_RESPONSE),
+    ChatRouteEnum.coach: ChatPromptTemplate.from_template(COACH_RESPONSE),
+    ChatRouteEnum.gratitude: ChatPromptTemplate.from_template("You're welcome!"),
+    ChatRouteEnum.summarisation: ChatPromptTemplate.from_template(
+        "You are asking for summarisation - route not yet implemented"
+    ),
+    ChatRouteEnum.extract: ChatPromptTemplate.from_template(
+        "You asking to extract some information - route not yet implemented"
+    ),
```

Comment:
I like the `ChatRouteEnum` class. Any objection to renaming it `ChatRoute`, which sounds a bit cleaner to me?